upup/models/cloudup/resources/addons/networking.projectcalico.org.canal/k8s-1.9.yaml.template

```diff
@@ -0,0 +1,543 @@
+# Canal Version v3.2.3
+# https://docs.projectcalico.org/v3.2/releases#v3.2.3
+# This manifest includes the following component versions:
+#   calico/node:v3.2.3
+#   calico/cni:v3.2.3
+#   coreos/flannel:v0.9.1
```

Comment:
Strange .. because something has changed in canal-node as i've checked the FORWARD (i.e. our blocking rules) chain is being observed. Also, I think we can drop those and use host protection now .. https://docs.projectcalico.org/v3.2/getting-started/bare-metal/policy/

Reply:
Yeah we should drop in favour of the host policies, it's just a potential breaking change on upgrade for users (making use of `FELIX_CHAININSERTMODE`), until they've had time to switch to the global policies. I'll have to test and see if it's still a problem with this PR (related issue: https://github.com/coreos/flannel/issues/938).

Reply:
```shell
core@ip-10-250-102-81 ~ $ sudo iptables -L FORWARD -nv 
Chain FORWARD (policy ACCEPT 0 packets, 0 bytes)
 pkts bytes target     prot opt in     out     source               destination         
   38  6382 KUBE-FORWARD  all  --  *      *       0.0.0.0/0            0.0.0.0/0            /* kubernetes forwarding rules */
    0     0 REJECT     tcp  --  cali+  *       0.0.0.0/0            169.254.169.254      tcp dpt:80 state NEW /* AWS metadata endpoint */ reject-with icmp-port-unreachable
    0     0 REJECT     tcp  --  cali+  *       0.0.0.0/0            10.10.0.0/16         tcp dpt:22 state NEW /* SSH */ reject-with icmp-port-unreachable
    0     0 REJECT     tcp  --  cali+  *       0.0.0.0/0            10.10.0.0/16         tcp dpts:2380:2381 state NEW /* ETCD Peering */ reject-with icmp-port-unreachable
    0     0 REJECT     tcp  --  cali+  *       0.0.0.0/0            10.10.0.0/16         tcp dpts:4001:4002 state NEW /* ETCD API */ reject-with icmp-port-unreachable
    0     0 REJECT     tcp  --  cali+  *       0.0.0.0/0            10.10.0.0/16         tcp dpt:4194 state NEW /* cAdvisor */ reject-with icmp-port-unreachable
    0     0 REJECT     tcp  --  cali+  *       0.0.0.0/0            10.10.0.0/16         tcp dpt:8989 state NEW /* Kube Auth Listener */ reject-with icmp-port-unreachable
    0     0 REJECT     tcp  --  cali+  *       0.0.0.0/0            10.10.0.0/16         tcp dpt:9091 state NEW /* Calico Felix Prometheus Metrics */ reject-with icmp-port-unreachable
    0     0 REJECT     tcp  --  cali+  *       0.0.0.0/0            10.10.0.0/16         tcp dpt:9099 state NEW /* Calico health check */ reject-with icmp-port-unreachable
    0     0 REJECT     tcp  --  cali+  *       0.0.0.0/0            10.10.0.0/16         tcp dpts:10250:10256 state NEW /* kube http services */ reject-with icmp-port-unreachable
    0     0 REJECT     tcp  --  cali+  *       0.0.0.0/0            10.250.0.0/16        tcp dpt:22 state NEW /* SSH */ reject-with icmp-port-unreachable
    0     0 REJECT     tcp  --  cali+  *       0.0.0.0/0            10.250.0.0/16        tcp dpts:2380:2381 state NEW /* ETCD Peering */ reject-with icmp-port-unreachable
    0     0 REJECT     tcp  --  cali+  *       0.0.0.0/0            10.250.0.0/16        tcp dpts:4001:4002 state NEW /* ETCD API */ reject-with icmp-port-unreachable
    0     0 REJECT     tcp  --  cali+  *       0.0.0.0/0            10.250.0.0/16        tcp dpt:4194 state NEW /* cAdvisor */ reject-with icmp-port-unreachable
    0     0 REJECT     tcp  --  cali+  *       0.0.0.0/0            10.250.0.0/16        tcp dpt:8989 state NEW /* Kube Auth Listener */ reject-with icmp-port-unreachable
    0     0 REJECT     tcp  --  cali+  *       0.0.0.0/0            10.250.0.0/16        tcp dpt:9091 state NEW /* Calico Felix Prometheus Metrics */ reject-with icmp-port-unreachable
    0     0 REJECT     tcp  --  cali+  *       0.0.0.0/0            10.250.0.0/16        tcp dpt:9099 state NEW /* Calico health check */ reject-with icmp-port-unreachable
    0     0 REJECT     tcp  --  cali+  *       0.0.0.0/0            10.250.0.0/16        tcp dpts:10250:10256 state NEW /* kube http services */ reject-with icmp-port-unreachable
    0     0 REJECT     tcp  --  cali+  *       0.0.0.0/0            10.250.0.0/16        tcp dpts:30000:32767 state NEW /* nodeport range */ reject-with icmp-port-unreachable
    0     0 cali-FORWARD  all  --  *      *       0.0.0.0/0            0.0.0.0/0            /* cali:wUHhoiAYhphO9Mso */
```

Also curl the 10250 from Pod and watched the rules get tiggered
```shell
 22  1320 REJECT     tcp  --  cali+  *       0.0.0.0/0            10.250.0.0/16        tcp dpts:10250:10256 state NEW /* kube http services */ reject-with icmp-port-unreachable
```